test/box/gh-5928-several-listen-sockets.test.lua

```diff
@@ -0,0 +1,246 @@
+env = require('test_run')
+net_box = require('net.box')
+fio = require('fio')
+fiber = require('fiber')
+test_run = env.new()
+test_run:cmd("create server test with script='box/gh-5928-several-listen-sockets.lua'")
+
+test_run:cmd("setopt delimiter ';'")
+function str_split (inputstr, sep)
+        local t = {}
+        if string.find(inputstr, sep) == nil then
+            t[#t + 1] = inputstr
+            return t
+        end
+        for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
+                t[#t + 1] = str
+        end
+        return t
+end;
+function check_connection(server_addr)
+     local conn = net_box.new(server_addr)
+     if not conn then
+     	return false
+     end
+     local rc = true
+     if conn:ping() == false then
+         rc = false
+     end
+     conn:close()
+     return rc
+end;
+function prepare_several_listen_uri(default_server_addr, listen_sockets_count)
+    local listen = ""
+    for i = 1, listen_sockets_count do
+        local number = 97 + i - 1
```

Comment:
Do you mean port? Or what this number is supposed to mean?

Reply:
This is ascii code of 'a' so if default_server_addr '/tmp/1' (for example, this function and listen_sockets_count == 3 returns 
"/tmp/1A,  /tmp/1B,  /tmp1C"